ruby/engine/RubyEngine.cpp

```diff
@@ -41,7 +43,10 @@ VALUE initRestOfOpenStudio(...) {
 };
 
 RubyEngine::RubyEngine(int argc, char* argv[]) : ScriptEngine(argc, argv) {
+
+  ruby_set_argv(argc, argv);
```

Comment:
On windows, ruby_sysinit(&argc, &argv) was *somehow* picking up the full path to openstudio.exe even though we init the RubyEngine by poping it (the first argv). This fixes it